exercises/container-security/docker-hub-cli/cli/index.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+const args = require('minimist')(process.argv.slice(2));
+const chalk = require('chalk');
+const dockerMetadata = require('../lib');
```

Comment:
`dockerMetadata` is a bit confusing, since its adding new name to the lib code (not in lib source). Perhaps just `lib` or directly expand to `getDockerContentDigest` function?